src/modules/hotkeys/i18n/en_AU.root.json

```diff
@@ -130,6 +130,7 @@
                 "onlyAmbulance": "Backalarm only ambulances",
                 "title": "Backalarm"
             },
+            "easter_click": "Collect Easter Egg",
```

Comment:
I'm not yet sure if using the "Easter egg"-Term only.
Maybe we can find something general without listing each kind (sunflower, pumpkin, snowman, heart).

How about something like "Collect Easter Egg (or similar collective)"?
For DE I could think of using "Sammelobjekt (z.B. Osterei) einsammeln".

But I'm not yet sure. Maybe there are further ideas / opinions @LSS-Manager/developers ?

Reply:
How about Collect Seasonal Items? It's a fair point it doesn't signal well I guess that it works on other collectibles.

Reply:
Requested a review by accident - mistook the button for a refresh button 🤦🏻‍♂️

Reply:
Agree, we should mention an example. Especially for new players, which never had an event.

Reply:
I am fine with "Collect Seasonal Items e.g. Easter Egg"!
de_DE: "Saisonales Sammelobjekt einsammeln (z.B. Osterei)"